app/editor/src/features/content/form/ContentStoryForm.tsx

```diff
@@ -229,7 +229,7 @@ export const ContentStoryForm: React.FC<IContentStoryFormProps> = ({
         </Col>
       </Row>
       <Row className={contentType !== ContentTypeName.Image ? 'multi-section' : ''}>
-        <Show visible={contentType === ContentTypeName.Snippet}>
+        <Show visible={contentType === ContentTypeName.Snippet || showExpandModal}>
```

Comment:
I don't understand the logic here?  Why are we showing the TimeLogSection when we want to show the WYWISYG modal?

Reply:
Updated and good catch, thanks